shopfloor/services/service.py

```diff
@@ -28,6 +28,9 @@ def picking_type(self):
         """
         Get the current picking type based on the menu and the warehouse of the profile.
         """
+        # TODO: this validation should be done in the service
+        # and should return a proper error for the client
+        # or the client should handle exceptions properly.
```

Comment:
With my latest changes, the picking types are directly related to the menu, and the menus that do not have a picking type of the profile's warehouse will not be visible, so it will be solved.